tests/smoke/test.cli.build.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import {spawn} from 'child_process';
+import {describe, it} from 'mocha';
+
+import {
+  webExt, addonPath, artifactsPath,
+} from './common';
+
+describe('web-ext build', () => {
+  it('webext build --source-dir SRCDIR --artifacts-dir OUTDIR', () => {
```

Comment:
if it makes more sense to add the full command line being tested then I'd suggest a few things:
* prefixing it with the command is redundant
* you could still adhere to a readable BDD descriptor like it('accepts: --source-dir SRCDIR --artifacts-dir OUTDIR')